json/src/transaction.rs

```diff
@@ -36,11 +36,14 @@ pub struct Transaction {
 	/// Value.
 	pub value: Uint,
 	/// R.
-	pub r: Uint,
+	#[serde(default)]
+	pub r: MaybeEmpty<Uint>,
 	/// S.
-	pub s: Uint,
+	#[serde(default)]
+	pub s: MaybeEmpty<Uint>,
 	/// V.
-	pub v: Uint,
+	#[serde(default)]
+	pub v: MaybeEmpty<Uint>,
```

Comment:
This code is used not only in tests, right? Not sure if this is the right place for a fix.